share/spice/dogo_movies/dogo_movies.js

```diff
@@ -0,0 +1,60 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_dogo_movies = function(api_result) {
+        if (!api_result || !api_result.results || !api_result.results.length) {
+            return Spice.failed('dogo_movies');
+        }
+
+        // Get original query.
+        var script = $('[src*="/js/spice/dogo_movies/"]')[0],
+            source = $(script).attr("src"),
+            query = decodeURIComponent(source.match(/dogo_movies\/([^\/]+)/)[1]);
+
+        Spice.add({
+            id: 'dogo_movies',
+            name: 'Kids Movies',
+            data: api_result.results,
+            meta: {
+                sourceName: 'DOGOmovies',
+                sourceUrl: 'http://www.dogomovies.com/search?query=' + encodeURIComponent(query) + '&ref=ddg',
+                itemType: 'kids movies'
+            },
+            normalize: function(item) {
+                var thumb = item.thumb;
```

Comment:
All image URL's get passed to our internal proxy function so the requests are made to DDG instead of a 3rd party, in order to protect our user's privacy. 

It looks like these image urls are breaking our (admittedly naive) image URL check, for now please append "http:" to the url so it get's proxied correctly:

```
var thumb = "http:" + item.thumb
```